components/progress-rope/examples/00-default.js

```diff
@@ -0,0 +1,41 @@
+/** @jsx jsx */
+
+import { GEL, jsx } from '@westpac/core';
+import { Grid, Cell } from '@westpac/grid';
+import { Button } from '@westpac/Button';
+import { ProgressRope, Item } from '../src';
```

Comment:
Can you use the absolute import here please. It's better for readability and docs
```suggestion
import { ProgressRope, Item } from '@westpac/progress-rope';
```